src/lib/converter/plugins/CommentPlugin.ts

```diff
@@ -303,7 +303,11 @@ export class CommentPlugin extends ConverterComponent {
                     if (!childComment) {
                         childComment = signature.comment = new Comment();
                     }
-                    childComment.copyFrom(comment);
+
+                    childComment.shortText = childComment.shortText || comment.shortText;
+                    childComment.text      = childComment.text      || comment.text;
+                    childComment.returns   = childComment.returns   || comment.returns;
+                    childComment.tags      = childComment.tags      || comment.tags;
```

Comment:
The first fix using `copyFrom` looks like the right solution to me here. Am I missing something for why it won't work and we need to potentially share references to tags?

Thanks for doing the digging to figure out why this wasn't working!

Reply:
Umm there _was_ a reason, but I’ve forgotten what it was 😂. It was late last night. If I get a chance tonight I’ll try the other way again and run the tests, and report back what happens. Thanks!

Reply:
Ok I remember what it was now. `copyFrom` overwrites everything in the child comment that may already be there, whereas doing it the other way preserves what's in the child comment unless those fields are falsy, in which case it takes whatever is in the parent.

Reply:
Not having as much in-depth knowledge about the ramifications, I wonder if modifying the `copyFrom` method to do this "conditional copying" instead might be a clean way to do it? It looks like it's only used in 2 places in the code.

Reply:
Ah, that makes sense, thanks! Looks good to me.